examples/java/pom.xml

```diff
@@ -283,6 +286,33 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-runners-flink_2.10</artifactId>
+      <version>${project.version}</version>
+      <scope>runtime</scope>
+    </dependency>
+
+    <dependency>
```

Comment:
Okay. I'll create two new PR separately. Flink runner will work only adding flink dependency. The rest changes is more Spark related, so I'll put them into another one.